wpilibjExamples/src/main/java/edu/wpi/first/wpilibj/examples/drivedistanceoffboard/subsystems/DriveSubsystem.java

```diff
@@ -19,12 +19,15 @@ public class DriveSubsystem extends SubsystemBase {
   private final ExampleSmartMotorController m_leftFollower =
       new ExampleSmartMotorController(DriveConstants.kLeftMotor2Port);
 
+  // The motors on the right side of the drive.
   private final ExampleSmartMotorController m_rightLeader =
       new ExampleSmartMotorController(DriveConstants.kRightMotor1Port);
 
   private final ExampleSmartMotorController m_rightFollower =
       new ExampleSmartMotorController(DriveConstants.kRightMotor2Port);
 
+  // The feedforward controller (note that these are example values only - DO NOT USE THESE FOR YOUR OWN ROBOT!)
```

Comment:
This is a little confusing because the actual values are in Constants- I think referring to Constants and removing the DO NOT USE warning would be better